Jenkinsfile

```diff
@@ -55,7 +55,6 @@ for (int i = 0; i < platforms.size(); ++i) {
 /* Execute our platforms in parallel */
 parallel(branches)
 
-
 stage('Verify demos')
```

Comment:
Unrelated but given we now have automated tests, should we removed this?

Reply:
I have no strong opinion. To keep this does not hurt, but it will make the builder take longer

Reply:
Yes, that's my reason actually.